### PR TITLE
Add compressx extension

### PR DESCRIPTION
**Pull request by dqhieu**
## Description

[CompressX](https://hieudinh.com/compressx) is a macOS application that helps compress video to reduce file size. This extension allow user to select a video in Finder and then run a Raycast command to compress the video with CompressX automatically

## Screencast

https://github.com/raycast/extensions/assets/8291457/442d5f52-499f-4ed2-9164-54a80ccc3315

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

File: extensions/compressx/src/compressVideo.tsx [hunk 1]
```diff
@@ -0,0 +1,25 @@
+import { showToast, Toast, getSelectedFinderItems } from "@raycast/api";
+import { exec } from "child_process";
+
+export default async function main() {
+  let filePaths: string[];
+
```
Comment (pernielsentikaer) [inline extensions/compressx/src/compressVideo.tsx:6]: Maybe you can check if it's installed? Extensions like [Eagle](https://www.raycast.com/Yukai/eagle) does that

Comment (dqhieu) [reply extensions/compressx/src/compressVideo.tsx:6]: Update with installation check. Can you help to review again? Thanks

**Review by pernielsentikaer — APPROVED** [review]
Hi 👋 

Looks good to me, approved 🔥

**Comment by dqhieu** [comment]
GitHub account has been linked to Raycast. Thanks

**Pull request merged**